src/platform/qt/test/library.cpp

```diff
@@ -0,0 +1,200 @@
+/* Copyright (c) 2013-2022 Jeffrey Pfau
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+#include "suite.h"
+#include "platform/qt/library/LibraryModel.h"
+
+#if QT_VERSION >= QT_VERSION_CHECK(5, 11, 0)
+#include <QAbstractItemModelTester>
+#endif
+
+#include <QSignalSpy>
+
+#define FIND_GBA_ROW(gba, gb) \
+	int gba = findGbaRow(); \
+	if (gba < 0) QFAIL("Could not find gba row"); \
+	int gb = 1 - gba;
+
+using namespace QGBA;
+
+class LibraryModelTest : public QObject {
+Q_OBJECT
+
+private:
+	LibraryModel* model = nullptr;
+
+	int findGbaRow() {
```

Comment:
_Extremely_ minor nit, but if you have to push anything might as well fix it: `findGBARow` is preferred here.

Reply:
Noted. (That disagrees with the Qt style guide, which uses e.g. `QHttpHeader`, so this was going to be a clash of style guides one way or the other.)